envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -391,11 +391,11 @@ abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
 
     @AfterEach
     fun cleanupTest() {
-        deregisterAllServices()
         envoyContainer1.admin().resetCounters()
         if (envoys == 2) {
             envoyContainer2.admin().resetCounters()
         }
+        deregisterAllServices()
```

Comment:
Why did you change the order of method calls?

Reply:
It seemed to me that it takes longer to rest counters than to deregister All services (our tests that were based on counters were flapping). I can revert it back.